devito/types/sparse.py

```diff
@@ -1148,19 +1148,19 @@ class PrecomputedSparseTimeFunction(AbstractSparseTimeFunction,
         So for `r=6`, we will store 18 coefficients per sparse point (instead of
         potentially 216).  Must be a three-dimensional array of shape
         `(npoint, grid.ndim, r)`.
-    space_order : int, optional
+    space_order : int, optional, default=0
         Discretisation order for space derivatives. Defaults to 0.
```

Comment:
Now all the "Defaults to ..." in the subsequent lines become redundant so they should maybe be dropped. There are quite a few